src/Parser/Core/Modules/Haste.js

```diff
@@ -37,6 +37,7 @@ class Haste extends Analyzer {
     [SPELLS.VOIDFORM_BUFF.id]: 0.01,
     [SPELLS.ICY_VEINS.id]: 0.3,
     [SPELLS.BONE_SHIELD.id]: 0.1, // Blood BK haste buff from maintaining boneshield
+    [SPELLS.MIND_QUICKENING.id]: (800 / 37500), // shadow priest trait
```

Comment:
Because this is a rating bonus and not a haste percent bonus, it's handled instead in StatTracker.

Reply:
Alright, I wondered a bit about stat bonuses. I will adjust.